src/Microsoft.DotNet.Darc/DarcLib/Helpers/ProcessManager.cs

```diff
@@ -76,10 +78,18 @@ public Task<ProcessExecutionResult> ExecuteGit(string repoPath, string[] argumen
             processStartInfo.ArgumentList.Add(arg);
         }
 
-        _logger.LogDebug("Executing command: '{executable} {arguments}'{workingDir}",
-            executable,
-            string.Join(' ', processStartInfo.ArgumentList),
-            workingDir is null ? string.Empty : " in " + workingDir);
+        var logMessage = $"Executing command: '{executable} {string.Join(' ', processStartInfo.ArgumentList)}'" +
```

Comment:
What's the point of such structure `$"..." + $""` with complex logic inside. I'd suggest something like
```C#
var logMessage = $"Executing command: {executable} "
    + string.Join(' ', processStartInfo.ArgumentList)
    + (workingDir is null ? string.Empty : " in " + workingDir);
```
From my point of view it's more readable

Reply:
Yeah, good idea